python/ecl/util/util/install_abort_signals.py

```diff
@@ -12,6 +13,11 @@ def updateAbortSignals():
     """
     Will install the util_abort_signal for all UNMODIFIED signals.
     """
-    if sys.version_info.major < 3 and not os.getenv('ECL_SKIP_SIGNAL'):
+    if sys.version_info.major < 3 and not os.getenv("ECL_SKIP_SIGNAL"):
         update_signals = EclPrototype("void util_update_signals()")
         update_signals()
+
+
+def setSignalOnUtilAbort(sig):
```

Comment:
snake case

Reply:
Shouldn't it be consistent with how the file already is?